server/src/service-layer/response-models/PaymentResponse.ts

```diff
@@ -14,6 +14,18 @@ export interface MembershipPaymentResponse extends CommonResponse {
   membershipType?: MembershipTypeValues
 }
 
+// Make a data shape matching to the expected response from Stripe API
+export interface MembershipStripeProductResponse extends CommonResponse {
+  data?: {
+    productId: string
+    name: MembershipTypeValues
+    description?: string
+    discount: boolean
+    displayPrice: string
+    originalPrice?: string
+  }
```

Comment:
```suggestion
  }[]
```
This needs to be an array